pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -50,8 +51,8 @@ in stdenv.mkDerivation rec {
       --set-rpath "${libPath}" \
       opt/${vivaldiName}/vivaldi-bin
   '' + stdenv.lib.optionalString proprietaryCodecs ''
-    sed -i '/^VIVALDI_FFMPEG_FOUND/ a \
-    checkffmpeg "${vivaldi-ffmpeg-codecs}/lib/libffmpeg.so"' opt/${vivaldiName}/vivaldi
+    sed -i '/^if \[ "$VIVALDI_FFMPEG_FOUND/i \
+      VIVALDI_FFMPEG_FOUND=YES\nCACHED_FFMPEG=${vivaldi-ffmpeg-codecs}/lib/libffmpeg.so' opt/${vivaldiName}/vivaldi
```

Comment:
`opt/${vivaldiName}/vivaldi` -> `opt/${vivaldiName}/${vivaldiName}` for compatibility with snapshots. Or fixing the symlink from `opt/${vivaldiName}/${vivaldiName}` to `opt/${vivaldiName}/vivaldi`.